app/enterprise/0.33-x/admin-gui/configuration/authentication.md

```diff
@@ -21,79 +21,147 @@ chapter: 3
 
+```bash
+curl -X POST http://localhost:8001/rbac/users/98af4bda-8525-44a6-a745-0c503b146639/roles -d 'roles=super-admin'
+
+{
+  "roles": [
+    {
+      "comment": "Default user role generated for user-myadmin",
+      "created_at": 1537966410000,
+      "id": "63be0c50-1a12-4483-986a-92bc5d3302b3",
+      "name": "user-myadmin"
+    },
+    {
+      "comment": "Full access to all endpoints, across all workspaces",
+      "created_at": 1537965959000,
+      "id": "5be18384-ff74-4242-96a6-36d5fa85af55",
+      "name": "super-admin"
+    }
+  ],
+  "user": {
+    "comment": "User generated on creation of Admin.",
+    "user_token": "406ee936-4a10-41c0-a9e5-7b9f7b6fc106",
+    "id": "98af4bda-8525-44a6-a745-0c503b146639",
+    "name": "user-myadmin",
+    "enabled": true,
+    "created_at": 1537966410000
+  }
+}
 ```
-curl -X POST http://kong:8001/rbac/users/{username}/roles/    \
-    --data "roles=super-admin"
-```
 
-Now that you have an Admin with an associated login Key and an RBAC Super-Admin, 
-update the following in your Kong Configuration, then restart Kong.
+Third, you must provision a Key-Auth api key for the Admin user's "hidden consumer":
+
+Note, in this example we are using key-auth; for other auth methods you will
+need to provisions the approprate key/password needed for that method.
 
+```bash
+curl -X POST http://localhost:8001/consumers/94110df6-211a-4f89-b4f3-0a994753ddf9/key-auth -d ''
+
+{
+  "created_at": 1537966734000,
+  "id": "ea20b2c6-a1c8-4f2e-8cbe-97f802506b6e",
+  "key": "Au5sciJOadbGyDAk6ZqndKH8IQFzKZ5x",
+  "consumer_id": "94110df6-211a-4f89-b4f3-0a994753ddf9"
+}
 ```
-admin_gui_auth = key-auth
+
+Save this key `Au5sciJOadbGyDAk6ZqndKH8IQFzKZ5x` for later, it is used at the
+Admin GUI login prompt. Note that this key is separate and distinct from the
+admin's RBAC token. 
+
+
+Forth, update the config to enable RBAC and the Admin GUI auth method:
+
+Using the kong.conf file:
+
+```bash
 enforce_rbac = on
+admin_gui_auth = key-auth
 ```
 
-The Admin GUI is now aware that authentication is enabled and will restrict access so that
-only the Super Admin can login. Browse to the Admin GUI and you will be prompted with a 
-[Login](#logging-in) form. Enter in the key saved from before to gain access to the Admin 
-GUI. You should be able to access all resources.
+Using environmental variables:
+
+```bash
+KONG_ENFORCE_RBAC = on
+KONG_ADMIN_GUI_AUTH = key-auth
+```
+
+Fifth, restart Kong.
+
+The Admin GUI is now aware that authentication is enabled and will restrict
+access so that only the Admin users with an api key and in the RBAC
+`super-admin` group can log in. Browse to the Admin GUI and you will be
+prompted with a [Login](#logging-in) form. Enter in the api key saved from
+before to gain access to the Admin GUI. You should be able to access all
+resources.
```

Comment:
suggestion: `...access all resources and provision new Admins from the GUI itself.`